source/standards/operating-systems.html.md.erb

```diff
@@ -1,34 +1,18 @@
 ---
-title: Operating Systems
+title: How to choose operating systems
 expires: 2018-01-31
 ---
 
 # <%= current_page.data.title %>
 
 <%= partial :expires %>
 
-When you operate a server, you need to choose an operating system for it.  This guidance makes recommendations for choosing an operating system for a Virtual Machine (VM).
+When choosing an operating system (OS) for your Virtual Machine (VM) we recommend using [Ubuntu Long Term Support (LTS)](https://www.ubuntu.com/download/desktop).
```

Comment:
Surely https://www.ubuntu.com/download/server in the case of a server?

Reply:
Of course, thank you. URL now changed!